docs-src/clusters/how-to-set-up-mysql-visibility-store.md

```diff
@@ -35,10 +35,10 @@ persistence:
       #...
     mysql-visibility:
       sql:
-        pluginName: "mysql" # if using MySQL 8.0.17 or later with Temporal Server v1.20, use "mysql8" plugin for advanced Visibility capabilities
+        pluginName: "mysql8" # For MySQL v8.0.7 and later. If using MySQL 5.7 or earlier, use "mysql" plugin.
```

Comment:
"5.7 or earlier" sounds like versions before 5.7, not before 8.0.17.

```suggestion
        pluginName: "mysql8" # For MySQL v8.0.17 and later. If using MySQL <8.0.17 or 5.7, use "mysql" plugin.
```

Reply:
Updated to "For MySQL v8.0.7 and later. For earlier versions, use "mysql" plugin."